docs/initiatives/suggestions.rst

```diff
@@ -0,0 +1,74 @@
+===============================
+Suggestions for LFN Initiatives
+===============================
+
+Update Infra
+------------
+
+LFN Projects are using a Gerrit+Jenkins+Nexus infra that is both antequated and
+expensive to maintain.  More modern open source projects tend to use Github based
+infra like:
+
+* Github - Source Code Management
+* CircleCI or TravisCI or other cloud based CI - CI
+* Various artifact repo as a service services
+* Github Issues - Bug tracking
+* Github + Hugo + Netlify - Websites/doc sites
```

Comment:
as mentioned in the discussion thread (https://github.com/lfnetworking/process/issues/1)
on our side we moved to gitlab because most of the features mentioned though github + .. are built-in.
we use the following features
- Source code management (very similar to github)
- CI/CD
- bug tracker
- website
- artifact management

Moreover gitlab is open source, which provides flexibility.

Reply:
@morganrOL Happy to throw gitlab into the consideration as well.  The thing I care most about personally is ecosystem of tools.  Github is the leader there, but gitlab is a close second (and has lots of other cool features as you mention).